docs/types/reference-types.rst

```diff
@@ -235,7 +235,7 @@ with the :ref:`default value<default-value>`.
         }
     }
 
-.. index:: ! array;literals, ! inline;arrays
+.. index:: ! literal;array, ! inline;arrays
```

Comment:
Is this change only to be consistent with the others?

Reply:
Yes. All literals now index entries under `literal` so there's no reason to single it out.

I could add both, but then I'd have to do that for the others too. I think this is enough.